src/views/Miscellaneous.vue

```diff
@@ -0,0 +1,153 @@
+              <ion-note slot="end">{{ getJobRuntime(job) }}</ion-note>
+            </ion-item>
+          </ion-list>
+
+          <ion-infinite-scroll @ionInfinite="loadMoreMiscellaneousJobs($event)" threshold="100px" :disabled="!isMiscellaneousJobsScrollable">
+            <ion-infinite-scroll-content loading-spinner="crescent" :loading-text="$t('Loading')"/>
+          </ion-infinite-scroll>
+        </section>
+
+        <aside class="desktop-only" v-if="isDesktop" v-show="currentJob && Object.keys(currentJob).length">
+          <JobConfiguration :title="currentJobTitle" :status="currentJobStatus" :key="currentJob"/>
+        </aside>
+      </main>
+    </ion-content>
+  </ion-page>      
+</template>
+
+<script lang="ts">
+import { DateTime } from 'luxon';
+import {
+  IonContent,
+  IonHeader,
+  IonInfiniteScroll,
+  IonInfiniteScrollContent,
+  IonItem,
+  IonLabel,
+  IonList,
+  IonListHeader,
+  IonMenuButton,
+  IonNote,
+  IonPage,
+  IonTitle,
+  IonToolbar,
+  isPlatform,
+} from '@ionic/vue';
+import { defineComponent } from 'vue';
+import { useRouter } from 'vue-router'
+import { mapGetters, useStore } from 'vuex'
+import emitter from '@/event-bus';
+import JobConfiguration from '@/components/JobConfiguration.vue';
+import { isFutureDate } from '@/utils';
+
+export default defineComponent({
+  name: 'Miscellaneous',
+  components: {
+    IonContent,
+    IonHeader,
+    IonInfiniteScroll,
+    IonInfiniteScrollContent,
+    IonItem,
+    IonLabel,
+    IonList,
+    IonListHeader,
+    IonMenuButton,
+    IonNote,
+    IonPage,
+    IonTitle,
+    IonToolbar,
+    JobConfiguration
+  },
+  mounted() {
+    this.getMiscellaneousJobs();
+  },
+  data() {
+    return {
+      currentJobTitle: '',
+      currentJobStatus: '',
+      isJobDetailAnimationCompleted: false,
+      isDesktop: isPlatform('desktop')
+    }
+  },
+  computed: {
+    ...mapGetters({
+      miscellaneousJobs: 'job/getMiscellaneousJobs',
+      getCurrentEComStore:'user/getCurrentEComStore',
+      currentJob: 'job/getCurrentJob',
+      isMiscellaneousJobsScrollable: 'job/isMiscellaneousJobsScrollable'
+    })
+  },
+  methods: {
+    async viewJobConfiguration(job: any) {
+      this.currentJob = job
+      this.currentJobTitle = job.jobName
+      this.currentJobStatus = job.status
+
+      // if job runTime is not a valid date then making runTime as empty
+      if (job?.runTime && !isFutureDate(job?.runTime)) {
+        job.runTime = ''
+      }
+
+      await this.store.dispatch('job/updateCurrentJob', { job });
+      if(!this.isDesktop && job?.jobId) {
+        this.router.push({name: 'JobDetails', params: { title: this.currentJobTitle, jobId: job?.jobId, category: "miscellaneous"}});
+        return;
+      }
+
+      if (job && !this.isJobDetailAnimationCompleted) {
+        emitter.emit('playAnimation');
+        this.isJobDetailAnimationCompleted = true;
+      }
+    },
+    async getMiscellaneousJobs(viewSize = 20, viewIndex = '0') {
```

Comment:
I think we could have viewSize and viewIndex both numbers

Reply:
Sure sir.